src/08_comprehensions.py

```diff
@@ -11,23 +11,26 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
-print (y)
+for x in range(6):
+    if x > 0:
+        y.append(x)
+print(y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
-
+for x in range(10):
+    y.append(x * x*x)
 print(y)
 
 # Write a list comprehension to produce the uppercase version of all the
 # elements in array a. Hint: "foo".upper() is "FOO".
 
 a = ["foo", "bar", "baz"]
-
 y = []
```

Comment:
```suggestion
y = [x.upper() for x in a]
```
list comprehensions!!!!